package.json

```diff
@@ -1,29 +1,20 @@
 {
   "private": true,
-  "name": "gatsby",
+  "name": "pancakeswap-api-v2",
   "version": "1.0.0",
   "dependencies": {
     "bignumber.js": "^9.0.1",
-    "gatsby": "^2.18.14",
-    "gatsby-image": "^2.0.15",
-    "gatsby-plugin-manifest": "^2.0.5",
-    "gatsby-plugin-react-helmet": "^3.0.0",
-    "react": "^16.5.1",
-    "react-dom": "^16.5.1",
-    "react-helmet": "^5.2.0",
     "web3": "^1.3.0"
   },
   "devDependencies": {
     "@types/jest": "^26.0.15",
     "@types/node": "^14.14.7",
-    "@now/node": "^1.3.0",
+    "@vercel/node": "^1.8.5",
     "jest": "^26.6.3",
     "ts-jest": "^26.4.4",
     "ts-node": "^9.0.0"
   },
   "scripts": {
-    "dev": "gatsby develop",
-    "build": "gatsby build",
```

Comment:
so what's the command for build then?

Reply:
You don't need a build command vercel is using the default nodejs api build command.
To build it on your local machine you can run vercel dev